libraries/standard/mqtt/src/iot_mqtt_api.c

```diff
@@ -267,6 +328,14 @@ static IotMqttError_t _waitForOperation( IotMqttOperation_t operation,
 
 /*-----------------------------------------------------------*/
 
+/**
+ * @brief Place holder packet identifier used when
+ * _IotMqtt_AddSubscriptions is called with previous subscriptions lists.
```

Comment:
I would add a comment clarifying if there are any restrictions on this value; why 2 and not 3?  If there are no restrictions then 0 might be a more obvious choice.

Reply:
Agree. There is no restriction and it is not used so 0 should work.  I'll add the clarification in the next PR, and update it to 0 if that works.